src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -1,16 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using Dynamo.Configuration;
+using Dynamo.Controls;
+using Dynamo.Logging;
+using Dynamo.ViewModels;
+using System;
 using System.Linq;
 using System.Windows;
 using System.Windows.Controls;
-using System.Windows.Data;
 using System.Windows.Documents;
 using System.Windows.Input;
 using System.Windows.Media;
-using Dynamo.Controls;
-using Dynamo.Logging;
-using Dynamo.Models;
-using Dynamo.ViewModels;
```

Comment:
These are not supposed to be touched, did you recently configured a new VS IDE? https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards

Reply:
@QilongTang I didn't touch the configurations of VS2019 but by some reason there was an option missing about "place System directives first" then I enable the setting and reorganized the usings again. see commit: https://github.com/DynamoDS/Dynamo/pull/12667/commits/5fc4b1e35ba1ad5b280d4b412fa73f762bed056e